### ISSUE TITLE
9.5 Staging 2180: "/skills give" cmd crashing local server

### ISSUE THREAD

**Issue by Mkorny** [StrangeLoopGames/EcoIssues#22857]
\
**User Id: (slg183077)**
**Version:** 0.9.5.0 beta staging-2180

**Steps to Reproduce:**
1. ran the cmd: /skills give logging
2. crash to main menu

**Expected behavior:**
should have learned the logging skill	

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Skills.SkillCommands.Give(User user, Type skillType)
   at Eco.Gameplay.Skills.SkillCommands.Give(User user, String skillName)
Outer Exceptions:
Exception has been thrown by the target of an invocation.
One or more errors occurred. (Exception has been thrown by the target of an invocation.)
Exception has been thrown by the target of an invocation.</size>
```

**Do you have mods installed? Does the issue happen wh

en no mods are installed?:**
no - running a simple local world.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/7751634/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/7751635/Player.log)

**Comment by TwixterZ** [comment]
Hi @Mkorny, thank you for raising this issue. I can confirm this is happening on build 2180, I have forward this to our developer attention.

**Comment by WeaselDog** [comment]
Thanks for your report. This will be fixed in the upcoming 9.5 release.
